### PR TITLE
Colmandrews/safety outcomes

### PR DESCRIPTION
**Pull request by andrewscolm**
Adds pericarditis and myocarditis tot he outcomes and childhood vaccinations to the matching variables.

Updates the childhood vaccinations variables as described by @chrisjwood16 [here](https://github.com/opensafely/codelist-development/issues/254#issuecomment-1547903488)

### CODE REVIEW

**Comment by andrewscolm** [comment]
@chrisjwood16 https://github.com/opensafely/vaccine-effectiveness-in-kids/pull/64/files#diff-0066008db621cf42c81ba1d77d1ffc61ba557d6eb737bc064eb68181516c5d6bR208-R292 may be of most interest to you.

@wjchulme currently this PR adds both MMR and a combined `vax_compliant` using the logic described [here](https://github.com/opensafely/codelist-development/issues/254#issuecomment-1547903488). We may want to use one or the other when we run it.

**Comment by chrisjwood16** [comment]
The vaccine logic looks sensible to me.

File: analysis/variables_matching.py [hunk 1]
```diff
@@ -190,9 +193,105 @@ def generate_matching_variables(baseline_date):
...
+        AND
+        (
+        type_DTaP_IPV_Hib_HepB
+        OR
+        type_DTaP_IPV_Hib
+        )
+        AND
+        type_Hib_MenC
+        AND
+        type_MMR
```
[the remark this answers is not in the document]
Comment (andrewscolm) [reply analysis/variables_matching.py]: Thanks Will, I have removed the MMR now

**Review by wjchulme — APPROVED** [review]
All good

**Pull request merged**